packages/image-to-png/readme.md

```diff
@@ -0,0 +1,28 @@
+# @creately/image-to-png
+
+> This will convert Interlace PNG, SVG, GIF, ICON, BMP format to PNG image.
+
+Support: All browsers and node.js
+
+## Install
+
+```
+$ npm install @creately/image-to-png
+```
+
+## Usage
+
+```js
+import { ImageToPng } from '@creately/image-to-png';
+
+const convertedImage = new ImageToPng().toPNG('Image Base64 string');
+
+console.log(convertedImage);
+//=> 'data:image/png;base64,....'
+```
```

Comment:
Since `toPNG` returns a promise, the console log here wouldn't log the resolved value. Better if the readme can be updated to something like
```
const imageToPng = new ImageToPng();
const convertedImage = await imageToPng.toPNG('Image Base64 string');
console.log(convertedImage);
//=> 'data:image/png;base64,....'
```